src/api/user/create/index.ts

```diff
@@ -0,0 +1,89 @@
+import { User } from "../../../ui/pages/usersList/types";
+import { getApiUrl, useFetchData } from "../../../utils";
+
+export type CreatePasswordlessUserPayload = {
+	email?: string;
+	phoneNumber?: string;
+};
+
```

Comment:
need to make sure that post creation of a user, the dev is shown the user details page in a new tab

Reply:
yep!